src/dp_flow.c

```diff
@@ -148,6 +192,10 @@ int dp_build_flow_key(struct flow_key *key /* out */, struct rte_mbuf *m /* in *
 	case IPPROTO_ICMP:
 		ret = dp_build_icmp_flow_key(df, key, m);
 		break;
+	case IPPROTO_ICMPV6:
+		key->port_dst = ntohs(df->l4_info.icmp_field.icmp_identifier);
+		key->src.type_src = df->l4_info.icmp_field.icmp_type;
+		return DP_OK;
```

Comment:
This should be a break to make it consistent for all l4 types

Reply:
Yeah but one of the compilers in CI/CD (I think it was clang) with high level warnings enabled was complaining "break" is unreachable code so the break was removed intentionally.

Reply:
Oh, then I guess let's make all cases do a `return` because of that.

Reply:
I corrected it differently.  Take a look after I push the commit.